src/components/List/List.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { serifRegular16, serifRegular19 } from '../Typography/styles'
+import colors from '../../theme/colors'
+import { css } from 'glamor'
+import { mUp } from '../../theme/mediaQueries'
+
+const SPACING = 22
+
+const unorderedBefore = {
+  '& > li:before': {
+    content: '–',
+    position: 'absolute',
+    left: 0,
+    [mUp]: {
+      left: `-${SPACING}px`
+    }
+  }
+}
+
+const orderedBefore = {
+  '& > li:before': {
+    content: 'counter(item) ". "',
+    counterIncrement: 'item',
+    position: 'absolute',
+    left: 0,
+    [mUp]: {
+      left: `-${SPACING}px`
+    }
+  }
+}
+
+const styles = {
+  list: css({
+    marginLeft: 0,
+    paddingLeft: 0,
+    listStyle: 'none',
+    counterReset: 'item',
+    '& li': {
+      color: colors.text,
+      margin: '24px 0',
+      paddingLeft: `${SPACING}px`,
+      position: 'relative',
+      ...serifRegular16,
+      [mUp]: {
+        ...serifRegular19,
+        paddingLeft: 0,
+        margin: '30px 0',
+        lineHeight: '30px'
+      }
+    },
+    '& li li': {
+      fontSize: '14px',
```

Comment:
Could we use `Typography/styles` mixing here? (book keeping of all sizes is good)